ocs_ci/ocs/utils.py

```diff
@@ -830,7 +830,15 @@ def collect_ocs_logs(dir_name, ocp=True, ocs=True, mcg=False):
             '/usr/bin/gather_service_logs worker'
         )
     if mcg:
-        collect_noobaa_db_dump(log_dir_path)
+        counter = 0
+        mcg_db_collected = False
+        while not mcg_db_collected or counter < 5:
+            try:
+                collect_noobaa_db_dump(log_dir_path)
+                mcg_db_collected = True
+            except CommandFailed:
+                counter += 1
```

Comment:
This ending up in infinite loop @ebenahar .

See this upgrade execution it got stuck in all the time collecting noobaa stuff :(

https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/9782/consoleFull

You should add counter += 1 at the start before line 836.

Reply:
Opened issue here:
https://github.com/red-hat-storage/ocs-ci/issues/2496#issue-656141735

Reply:
Fixing here: https://github.com/red-hat-storage/ocs-ci/pull/2497